metadata/provider_metadata.json

```diff
@@ -89380,7 +89380,7 @@
             },
             {
                 "name": "public_subnet",
-                "type": "TypeString",
+                "type": "TypeInt",
```

Comment:
This is a generated file.
We generate it as part of the release.
Kindly revert the changes on provider_metadata.json file

Reply:
@kavya498 I have reverted that change as requested.